standard/toolchain-arm64.cmake

```diff
@@ -1,13 +1,9 @@
 set(CMAKE_SYSTEM_NAME Linux)
-set(CMAKE_SYSTEM_PROCESSOR arm)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
```

Comment:
Yeah, probably good to opt for `aarch64`. From what I can tell OpenCV and libpng use `aarch64` while assimp uses `arm64`.